srtcore/buffer.h

```diff
@@ -227,7 +227,7 @@ class CSndBuffer
       int32_t m_iMsgNoBitset;           // message number
       int32_t m_iSeqNo;                 // sequence number for scheduling
       time_point m_tsOriginTime;        // original request time
-      uint64_t m_ullSourceTime_us;
+      int64_t m_ullSourceTime_us;
```

Comment:
Name should then also change to `m_llSourceTime_us`.

Reply:
Changed.